spec/models/payment_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper'
+
+describe Payment do
+
+  describe 'validations' do
+    it 'has a valid factory' do
+      expect(build(:payment)).to be_valid
+    end
+
+    it 'is not valid without a first_name' do
+      should validate_presence_of(:first_name)
+    end
+
+    it 'is not valid without a last_name' do
+      should validate_presence_of(:last_name)
+    end
+
+    it 'is not valid without a credit_card_number' do
+      should validate_presence_of(:credit_card_number)
+    end
+
+    it 'is not valid without a card_verification_value' do
+      should validate_presence_of(:card_verification_value)
+    end
+
+    it 'is not valid without a expiration_month' do
+      should validate_presence_of(:expiration_month)
+    end
+
+    it 'is not valid without a expiration_year' do
+      should validate_presence_of(:expiration_year)
+    end
+
+    it 'is not valid without a amount' do
+      should validate_presence_of(:amount)
+    end
+
+    it 'is not valid with a amount equals zero' do
+      should_not allow_value(0).for(:amount)
+    end
+
+    it 'is not valid with a amount smaller than zero' do
+      should_not allow_value(-1).for(:amount)
+    end
+
+    it 'is valid with a amount greater than zero' do
+      should allow_value(1).for(:amount)
+    end
```

Comment:
from line 10 until here, you can write the specs like:

```ruby
it { is_expected.to validate_presence_of(:last_name) }`
```

That way, you don't need so much repetition.